apps/ima/views/life-or-liberty-threatened.html

```diff
@@ -0,0 +1,15 @@
+{{<partials-page}}
+  {{$page-content}}
+    <p>Your life or liberty must have been threatened because of your:</p>
+    <ul class="list list-bullet">
+      <li>race</li>
+      <li>religion</li>
+      <li>nationality</li>
+      <li>membership of a social group</li>
+      <li>political opinions</li>
+    </ul>
+    <p>If you passed through or stopped in another country where your life or liberty was not threatened for these reasons, you did not come directly to the UK.</p>
```

Comment:
Cross check with the text on figma, they are not same. you can write the sentence in page.json and render here

Reply:
Amended the text, moved the sentence to the page.json file, and referenced it from there